.planning/phases/01-foundation/01-01-PLAN.md

```
@@ -0,0 +1,305 @@
+    - app/layout.tsx — read current font imports to know exactly what to remove
+    - tailwind.config.ts — read current fontFamily config to know what to strip
+    - app/globals.css — verify body font-family:Arial line (will fix in Plan 03)
+  </read_first>
+  <action>
+    Step 1 — Delete Geist font files. These are dead weight and MUST be removed BEFORE the Tailwind codemod runs in Plan 02 (so dead font tokens don't get carried forward into @theme {}):
+    ```bash
+    rm /Users/engr.mujeeburrahman/personal-project/Portfolio-1.1/app/fonts/GeistVF.woff
+    rm /Users/engr.mujeeburrahman/personal-project/Portfolio-1.1/app/fonts/GeistMonoVF.woff
+    ```
+
+    Step 2 — Update app/layout.tsx: Remove the `geistSans` and `geistMono` localFont() declarations and their CSS variable class names from `<body>`. Remove the `import localFont from 'next/font/local'` line entirely.
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)